components/fxa-client/src/internal/http_client.rs

```diff
@@ -53,37 +53,37 @@ const DEVICES_FILTER_DAYS: u64 = 21;
 /// it return something called a "refresh token"? Using unambiguous
 /// verbs to start each method helps avoid confusion here.
 ///
-#[cfg_attr(test, mockiato::mockable)]
+#[cfg_attr(test, mockall::automock)]
```

Comment:
Lets tell clippy to ignore the issue here, looks like we're hitting a limitation with mockall!
```suggestion
// Due to limitations in mockall, we have to add explicit lifetimes that the Rust compiler would have been happy to infer.
#[allow(clippy::needless_lifetimes)]
#[cfg_attr(test, mockall::automock)]
```